src/Workspaces/Core/Portable/Workspace/Host/TaskScheduler/IWorkspaceTaskScheduler.cs

```diff
@@ -1,44 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Diagnostics.CodeAnalysis;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Microsoft.CodeAnalysis.Host
-{
-    /// <summary>
-    /// An abstraction for running tasks either in sequence or in parallel.
-    /// </summary>
-    internal interface IWorkspaceTaskScheduler
```

Comment:
any chance this impacts VS4Mac?

Reply:
There is definitely a chance. I'm still waiting for them to send me list of internal APIs they use. In any case, they can easily change to TaskQueue as well.